pandas/core/internals/concat.py

```diff
@@ -471,21 +471,26 @@ def get_reindexed_values(self, empty_dtype: DtypeObj, upcasted_na) -> ArrayLike:
                     if len(values) and values[0] is None:
                         fill_value = None
 
-                if is_datetime64tz_dtype(empty_dtype):
+                elif is_datetime64tz_dtype(empty_dtype):
```

Comment:
this didn't need to change? (just a leftover from the code move?)

Reply:
yes, good catch